deployment/chainloop/README.md

```diff
@@ -311,7 +327,7 @@ controlplane:
 
 ### Use AWS secret manager
```

Comment:
AWS Secrets Manager - this is the name https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html

Reply:
Good catch!